test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -2145,6 +2151,13 @@ class test_ucp_sockaddr_protocols : public test_ucp_sockaddr {
                 rreq = do_unexp_recv(recv_buf, size, sreq, send_stop,
                                      recv_stop);
                 reqs.push_back(rreq);
+            } else {
+                ucs_status_t send_status = request_wait(sreq, 0, false,
+                                                        UCS_LOG_LEVEL_INFO);
+                if (!check_send_status(send_status, receiver(), NULL,
+                                       cb_type())) {
+                    break;
+                }
```

Comment:
but we already have requests_wait() below

Reply:
it may hang, if send fails in case of unreachability.